src/util/Enums.js

```diff
@@ -40,6 +40,12 @@ define({
   // Operations
   PRIMARY_AUTH: 'PRIMARY_AUTH',
   FORGOT_PASSWORD: 'FORGOT_PASSWORD',
-  UNLOCK_ACCOUNT: 'UNLOCK_ACCOUNT'
+  UNLOCK_ACCOUNT: 'UNLOCK_ACCOUNT',
+
+  // Auth Types
+  SESSION_STEP_UP: 'SESSION_STEP_UP',
```

Comment:
Can you check with @srinivasanagandla-okta whether we know that the normal mode is going to return an auth type of SESSION_SSO? If so, it'd be nice to include that now in this PR - if it's not currently being returned by the API, we could add it ourselves until it's there.